### PR TITLE
chore: add cli example

### PR DESCRIPTION
**Pull request by simoneb**
After addressing https://github.com/nearform/fastify-auth0-verify/pull/41 I wanted to make a release but I also wanted to make sure I didn't break anything. Before releasing, I thought I'd put together a CLI example which can be run to use the plugin and check that it works against a real instance of Auth0.

### CODE REVIEW

File: examples/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+  fastify.get(
+    '/',
+    {
+      preValidation: fastify.authenticate
+    },
+    r => r.user
+  )
+
+  await fastify.listen()
```
Comment (ShogunPanda) [inline examples/index.js:20]: I think listen is not needed if you're just using inject.

File: examples/index.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  console.log('\n==============\n|  Response  |\n==============\n')
+  console.log(JSON.stringify(JSON.parse(response.body), null, 2))
+
+  fastify.close()
+}
+
+run(
+  process.argv
+    .slice(2)
+    .map((o, i, _) => !(i % 2) && [o, _[i + 1]])
```
Comment (ShogunPanda) [inline examples/index.js:38]: This is at the same amazing and terrifing.
For better readability, shall we use something like minimist?

File: examples/index.js (hunk 1 above)
Comment (ShogunPanda) [reply examples/index.js:20]: Confirmed. I tested it locally. You can remove it.

Comment (simoneb) [reply examples/index.js:20]: You're right, done

File: examples/index.js (hunk 2 above)
Comment (simoneb) [reply examples/index.js:34]: I didn't want to add a dependency for the sake of a simple example. This is not code we're releasing

**Review by ShogunPanda — APPROVED** [review]
LGTM!

File: examples/index.js (hunk 2 above)
Comment (simoneb) [reply examples/index.js:34]: As discussed, I'll keep this to avoid introducing a dependency

**Pull request merged**